src/appmake/appmake.c

```diff
@@ -202,7 +202,7 @@ long parameter_search(const char *filen,const  char *ext,const char *target)
      /* Successfully opened the file so search through it.. */
     while ( fgets(buffer,LINEMAX,fp) != NULL ) {
         if (strncmp(buffer,target,strlen(target)) == 0 && isspace(buffer[strlen(target)])) {
-            sscanf(buffer,"%*s%*s%*[ $]%lx",&val);
+            sscanf(buffer,"%*s%*s%*[ $]%lx", (long unsigned int *) &val);
```

Comment:
See pull request text: sticking with a typecast for the pointer to val seems the most elegant and compatible with different C versions and all compilers.